### DISCUSSION TITLE
Question regarding level2 output and file creation/overwriting

### DISCUSSION THREAD

**Discussion by thielfab** [davidfrantz/force#288] [Q&A] [answered]
What is the default behavior of force-level2 with regard to its output if the file(s) already exists?

Due to #163 I wanted to reprocess some S2 scenes. I started with 1 image for test purposes. I assumed existing files will be overwritten, so I just ran force-l2ps with the default datacube settings. The modification time of the logfile and level2 outputs were updated, so it seemed the files are new. 
But, in the newly created (?) images the visual glitches (#163) were still present. If I run the level2 processing to a new/empty folder, the output looks fine.
Now I am confused.

**Comment by ernstste** [comment, marked as answer]
If there are files existing already, FORCE will create an average of the existing and new data.

Heads up: this is also relevant for the current reprocessing of Landsat 9 if you are running an operational processing system.

**Reply by thielfab** [reply]
Thanks! Now I can make sense of it. Never would have guessed that though.

**Reply by davidfrantz** [reply]
Here you go ;)

![image](https://github.com/davidfrantz/force/assets/32633788/513d2497-2560-4394-b23c-b2d721925dd4)

(Frantz 2019)
